docs/DeveloperGuide.md

```diff
@@ -639,13 +639,28 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Finding a person
+
+1. Finding with parameters missing.
+   1. Test case: `find` <br>
+      Expected: An error message is shown stating that the command format is incorrect and showing the user the correct format.
+   2. Test case: `find t/` <br>
+      Expected: Listing out everyone is the contact list.  
+2. Finding a person with all possible parameters specified.
```

Comment:
Should we add a Prerequisite: There is someone in the contact list with the name John Doe, phone number of 98765432, lives in room #08-0805 and has a "friends" tag here? Or alternatively, add another step to the test case, which is add n/John Doe p/98765432 e/johnd@example.com r/08-0805 t/friends